content/package.json

```diff
@@ -12,7 +12,9 @@
     "chokidar": "3.4.0",
     "dotenv": "8.2.0",
     "fast-csv": "4.1.3",
+    "got": "10.7.0",
```

Comment:
I suppose this is because of that last remaining Gecko version macro. Where is that anyway?

By the way, we should consider scripting the thing that clones the kumascript macros. And in that bash script we should filter out macros that we should definitely ignore. E.g.

```bash
#!/bin/bash

# Explanation here...

find $1/macros/*.ejs | grep -ve (DocStatus|OtherCrap)  | cp kumascript/macros/
```
@Gregoor started by creating a `docs/` folder. Can you create a `docs/kumascript.md` that explains the archtecture (lightly) and mention the script and some notes about the future?

Reply:
@peterbe Good idea. I'll add this to my TODO list, and create a separate PR once this lands in `master` if that's OK?